packages/survey-creator-core/src/components/tabs/logic-ui.ts

```diff
@@ -141,6 +142,7 @@ export class SurveyLogicUI extends SurveyLogic {
     super.onStartEditing();
     this.expressionEditorValue = this.getExpressionEditor(this.editableItem);
     this.itemEditorValue = this.getLogicItemEditor(this.editableItem);
+    this.expressionEditorIsFastEntry = false;
```

Comment:
@OlgaLarina This line switched into Builder if it is possible on start edting. We do not need it. I just wanted to hide/show action on toolbar. Show on startEditing and hide on cancelEditing, but I want to keep it state as it was.

Thank you,
Andrew

Reply:
Discussed by voice.